api/proto/info/info.proto

```diff
@@ -335,6 +335,9 @@ message ZInfoDevice {
 
   bool maintenance_mode = 38;
   MaintenanceModeReason maintenance_mode_reason = 39;
+
+  // Is /dev/watchdog present in the system
+  bool watchdog_present = 40;
```

Comment:
Since the API field name might be reflected in the UI, maybe we should call it hardware_watchdog_present?